docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+element` of current web page is saved using [`chrome.storage`](https://developer.chrome.com/extensions/storage) before
+the operation is performed. It is used for restoring focus on the same element when the user navigates back to the same
+page.
+
+### `{inputMapper.base}.nextPageInHistory(that, value)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Navigates to the next page in history when the gamepad `button` or `trigger` is pressed. It uses the
+[`History API`](https://developer.mozilla.org/en-US/docs/Web/API/History_API) for the core functionality. The `focused
+element` of current web page is saved using [`chrome.storage`](https://developer.chrome.com/extensions/storage) before
+the operation is performed. It is used for restoring focus on the same element if the user navigates back to the same
+page.
+
+### `{inputMapper.base}.thumbstickHistoryNavigation(that, value, invert)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `invert {Boolean}` Whether the thumbstick history navigation should be in opposite order.
+- Returns: Nothing.
+
+Calls the [`{inputMapper.base}.previousPageInHistory`](#inputmapperbasepreviouspageinhistorythat-value) and
+[`{inputMapper.base}.nextPageInHistory`](#inputmapperbasenextpageinhistorythat-value) invokers according to the
+direction the `thumbstick` is pressed. The direction can be inverted by passing `true` as the `invert` argument.
+
+### `{inputMapper.base}.reverseTab(that, value, direction)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `direction {String}` The direction for backward tab navigation, i.e. `reverseTab`.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the previous tabbable element using gamepad `buttons` and
+`triggers`. The previous tabbable element is the element before the currently focused element in the array stored as
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.forwardTab(that, value, direction)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `direction {String}` The direction for forward tab navigation, i.e. `forwardTab`.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the next tabbable element using gamepad `buttons` and
+`triggers`. The next tabbable element is the element next to the currently focused element in the array stored as
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.thumbstickTabbing(that, value, speedFactor, invert)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of tab navigation should be increased.
```

Comment:
This sentence can be improved, I suspect you should describe this in terms of frequency.  If there are units (for example, milliseconds), it's good to mention them.

Reply:
The `speedFactor` isn't frequency. It is currently used as a multiplication factor for the frequency and has no units.

Reply:
If I understand you correctly, you're multiplying the frequency (polling rate) by this?  If so, you should describe what is being scaled.  So, you might say "A scaling factor for the polling frequency, which affects the scrolling speed.  The default is to poll every X ms, setting this to less than 1 will slow down the scrolling, setting it to more than 1 will speed up the scrolling."  Something like that.